versions.yaml

```diff
@@ -9,6 +9,7 @@ module-sets:
       - go.opentelemetry.io/collector/pdata
       - go.opentelemetry.io/collector/config/configopaque
       - go.opentelemetry.io/collector/config/configcompression
+      - go.opentelemetry.io/collector/config/confignet
```

Comment:
It's funny how this module goes from 0.x not to 1.0 but straight to 1.3 :thinking:

Reply:
`1.0`, `1.1`, and `1.2` are overrated lol